packages/core/src/browser/quick-input/quick-access.ts

```diff
@@ -0,0 +1,75 @@
+/********************************************************************************
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company and others.
```

Comment:
RedHat?
or it was moved from another place?

Reply:
Yes, this started as "quick-access-contribution.ts". Let's not start a copyright war here.

Reply:
I didn't aware about `quick-access-contribution.ts`-  asked just in case - maybe you just missed that...  
anyway - it's not starting a copyright war :-)

Reply:
It's just that I had the same discussion with the SAP folks on the original monaco update ("don't change copyright on existing/renamed files"). :-)